src/app/features/asset-list/bitcoin/brc20-token-asset-list/brc20-token-asset-list.tsx

```diff
@@ -22,11 +23,27 @@ interface Brc20TokenAssetDetails {
 interface Brc20TokenAssetListProps {
   tokens: Brc20TokenAssetDetails[];
   variant?: AssetListVariant;
+  showBalance?: boolean;
+  renderRightElement?(id: string): ReactNode;
+  countSetter?: React.Dispatch<React.SetStateAction<number>>;
 }
-export function Brc20TokenAssetList({ tokens, variant }: Brc20TokenAssetListProps) {
+export function Brc20TokenAssetList({
+  tokens,
+  variant,
+  countSetter,
+  showBalance = true,
+  renderRightElement,
+}: Brc20TokenAssetListProps) {
+  useEffect(() => {
+    if (countSetter) {
+      countSetter(tokens.length);
+    }
+  }, [tokens, countSetter]);
   const navigate = useNavigate();
   const { balance, query: result } = useCurrentBtcCryptoAssetBalanceNativeSegwit();
 
+  if (!tokens.length) return null;
```

Comment:
inadvertently fixed issue #5469

the stack was creating the extra div even where there were no tokens

Reply:
Nice! How about adding that fix in a separate PR so we can get it released sooner?